tests/fixtures/projects.py

```diff
@@ -4,11 +4,17 @@
 from pontoon.base.models import Project
 
 
+@pytest.mark.env
```

Comment:
I don't understand this. Why are you marking a fixture?

Reply:
it was a failed attempt to use markers to automatically include in the `post_db_setup` fixture

Reply:
...ill remove